fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/recordrepair/RecordValidateAndRepairTest.java

```diff
@@ -138,18 +140,20 @@ void testValidateMissingSplit(int splitNumber, FormatVersion formatVersion, bool
                         ValidationTestUtils.assertInvalidResults(
                                 invalidResults,
                                 1,
-                                result -> result.equals(RecordRepairResult
-                                        .invalid(primaryKey, RecordRepairResult.CODE_SPLIT_ERROR, "any")
-                                        .withRepair(RecordRepairResult.REPAIR_RECORD_DELETED)));
+                                result -> !result.isValid() &&
+                                        result.getErrorCode().equals(RecordRepairResult.CODE_SPLIT_ERROR) &&
+                                        result.isRepaired() &&
+                                        result.getRepairCode().equals(RecordRepairResult.REPAIR_RECORD_DELETED));
+                        Assertions.assertThat((Object)primaryKey).isEqualTo(invalidResults.get(0).getPrimaryKey());
```

Comment:
You're mixing up `expected` and `actual` here
```suggestion
                        Assertions.assertThat(invalidResults.get(0).getPrimaryKey()).isEqualTo((Object)primaryKey);
```

Reply:
Oops. yeah - flipped.
There is some issue with the overlays of `assertThat` that gets ambiguous resolution error when calling it with `Tuple`, hence the cast to `Object`.